### PR TITLE
Idex 3851

### PR DESCRIPTION
**Pull request by olexii4**
@ashumilova WDYT?

### CODE REVIEW

File: commons/che-core-commons-gwt/src/main/java/org/eclipse/che/ide/ui/listbox/CustomListBox.java [hunk 1]
```diff
@@ -310,8 +310,10 @@ public void insertItem(String item, String value) {
     public void setHeight(String height) {
         this.getElement().getStyle().setProperty("height", height);
         final String lineHeightStyle = "line-height: " + height + ";";
+        boolean isVisible = optionsPanel.isVisible();
         currentItemLabel.setAttribute("style", lineHeightStyle);
-        optionsPanel.getElement().setAttribute("style", optionsPanel.isVisible() ? lineHeightStyle : lineHeightStyle + "display: none;");
+        optionsPanel.getElement().setAttribute("style", lineHeightStyle + ";max-height:calc(6*" + height + ");");
```
Comment (vzhukovskii) [inline commons/che-core-commons-gwt/src/main/java/org/eclipse/che/ide/ui/listbox/CustomListBox.java]: Don'n use such solutions. Element has style, you can work with it.

Comment (olexii4) [inline commons/che-core-commons-gwt/src/main/java/org/eclipse/che/ide/ui/listbox/CustomListBox.java]: done

**Comment by vzhukovskii** [comment]
+1 but move css colors to Theme

**Comment by ashumilova** [comment]
+1

**Pull request merged**